packages/material-ui/src/useTouchRipple/useTouchRipple.ts

```diff
@@ -25,7 +25,12 @@ interface RippleEventHandlers {
   onTouchStart: React.TouchEventHandler;
 }
 
-const useTouchRipple = (props: UseTouchRippleProps) => {
+const useTouchRipple = (
+  props: UseTouchRippleProps,
+): {
+  enableTouchRipple: boolean;
+  getRippleHandlers: (componentProps: Partial<RippleEventHandlers>) => RippleEventHandlers;
+} => {
```

Comment:
Explicit return types help inference and create more readable types. Before, TypeScript would show the whole object shape instead of re-using the `RippleEventHandlers` interface